app/src/main/java/net/gsantner/markor/activity/DocumentEditFragment.java

```diff
@@ -656,17 +657,28 @@ public void onFragmentFirstTimeVisible() {
         AppSettings as = new AppSettings(getContext());
         if (_savedInstanceState == null || !_savedInstanceState.containsKey(SAVESTATE_CURSOR_POS) && _hlEditor.length() > 0) {
             int lastPos;
-            if (_document != null && _document.getFile() != null && (lastPos = as.getLastEditPositionChar(_document.getFile())) >= 0 && lastPos <= _hlEditor.length()) {
-                if (!as.isPreviewFirst()) {
-                    _hlEditor.requestFocus();
+            if (as.getCursorPosition() != null) {
+                if (as.getCursorPosition().equals("End of document")) {
```

Comment:
This will break when translations are applied. You should do something like

```java
final Resources res = activity.getResources();
if (as.getCursorPosition().equals(res.getString(R.string.cursor_end)) {
```

Reply:
It would be even better, imo, to separate the string description from the key

Reply:
Yes that one was meant especially with "work in progress" :D

Reply:
Ah sorry, I am so used to looking for `WIP:` that I did not see the draft tag :)